config/webpack.config.js

```diff
@@ -153,6 +153,12 @@ const coreConfig = function() {
     'webpack/assets/javascripts/bundle.js'
   );
   config.context = path.resolve(__dirname, '..');
+  if (config.mode == 'production') {
```

Comment:
for some reason, this is always true, and I don't understand it.

Reply:
Looking at https://github.com/theforeman/foreman/blob/a837946894c6cf848efd444be3af76bd496dfa9b/config/webpack.config.js#L56-L59 we very often set `NODE_ENV` to `production`.

Reply:
https://github.com/theforeman/foreman/blob/fd21a88ca41dd15a9d35c40908c092280f610852/lib/tasks/webpack_compile.rake#L10

that could be it, indeed